docs: Update for `kwctl scaffold` (#99)
kubewarden/docs: src/tasks.md

```shell
$ kwctl --help
kwctl 0.2.5
Kubewarden Developers <kubewarden@suse.de>
Tool to manage Kubewarden policies

USAGE:
    kwctl [OPTIONS] <SUBCOMMAND>

OPTIONS:
    -h, --help       Print help information
    -v               Increase verbosity
    -V, --version    Print version information

SUBCOMMANDS:

>     annotate       Add Kubewarden metadata to a WebAssembly module

Reviewer: ```suggestion
    annotate       Adds Kubewarden metadata to a WebAssembly module
```

Author: While I get the points, just remember that this is the output from the `kwctl --help` command. Hence, we can't add link formatting, and the usual for commands is to have them in imperative voice (like in `git`). Some of the comments hold true, so I've opened PRs there when needed. For the rest, I would just leave it as it is in the docs.

>     completions    Generate shell completions

Reviewer: ```suggestion
    completions    Generates shell completions
```

>     digest         Fetch the digest of its OCI manifest

Reviewer: Better to clarify what “its” refers to here. I assume we’re referring to the OCI artefact containing the policy? In which case, we can rephrase it as the below.

```suggestion
    digest         Fetches the digest of the artifacts  OCI manifest
```

Author: Agree, opened https://github.com/kubewarden/kwctl/pull/189.

>     help           Print this message or the help of the given subcommand(s)

Reviewer: ```suggestion
    help           Prints this message or help for the given subcommand(s)
```

>     inspect        Inspect Kubewarden policy

Reviewer: ```suggestion
    inspect        Inspects Kubewarden policy
```

    policies       Lists all downloaded policies
    pull           Pulls a Kubewarden policy from a given URI
    push           Pushes a Kubewarden policy to an OCI registry
    rm             Removes a Kubewarden policy from the store
    run            Runs a Kubewarden policy from a given URI

>     scaffold       Scaffold a Kubernetes resource or configuration file

Reviewer: It’d be better to clarify which particular configuration file this scaffolds. 

- Is it ONLY a Kubernetes configuration file? 
- Is it any configuration file that can be scaffolded but passed as a separate parameter? 

```suggestion
    scaffold       Scaffolds a Kubernetes resource or configuration file
```

Author: Agree that it could be more verbose, but once one runs the command, it will show the subcommands:
```console
$ kwctl scaffold
error: 'kwctl scaffold' requires a subcommand but one was not provided

USAGE:
    kwctl scaffold <SUBCOMMAND>

For more information try --help
vic@viccuad2 2 ~$ kwctl scaffold --help
kwctl-scaffold
Scaffold a Kubernetes resource or configuration file

USAGE:
    kwctl scaffold <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    help                   Print this message or the help of the given subcommand(s)
    manifest               Output a Kubernetes resource manifest
    verification-config    Output a default Sigstore verification configuration file

```

Reviewer: Okay, then contextually it makes sense.

>     verify         Verify a Kubewarden policy from a given URI using Sigstore

Reviewer: ```suggestion
    verify         Verifies a Kubewarden policy from a given URI using [sigstore](https://github.com/sigstore)
```

Reviewer: I misunderstood the assignment, sorry. We can just change the verb and not include the link.

Author: I would prefer to not change the verbs and leave them in imperative voice, as other cli utilities do (e.g: `git`).

Reviewer: So, the rationale behind changing the voice was because this was the associated documentation. In the actual CLI utility, I don't recommend changing it.

Author: But, this is just a copy-paste from the command help output and stated as so in the docs, and we will refresh that output when we change commands in the future too (hence why I think it should match what a user gets when running the command help, and retouching it in the docs page is bound to always be out of sync with changes to the subcommand lists).
